### PR TITLE
Add CentOS support to the pnda build scripts

### PR DESCRIPTION
**Pull request by donaldh**

### CODE REVIEW

File: build/install-build-tools.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ if [[ -z $(grep `hostname` /etc/hosts) ]]; then
     exit -1
 fi
 
-if [[ "${DISTRO}" == "rhel" ]]; then
+if [ "x${DISTRO}" == "xrhel" -o "x$DISTRO" == "xcentos" ]; then
```
Comment (jeclarke) [inline build/install-build-tools.sh]: No need to prompt for RHEL subscription with centos, surely that's the whole point :-)

File: mirror/create_mirror.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ export DISTRO=$(cat /etc/*-release|grep ^ID\=|awk -F\= {'print $2'}|sed s/\"//g)
 [[ -z ${MIRROR_BUILD_DIR} ]] && export MIRROR_BUILD_DIR=${PWD}
 [[ -z ${MIRROR_OUTPUT_DIR} ]] && export MIRROR_OUTPUT_DIR=${PWD}/mirror-dist
 
-if [ "x$DISTRO" == "xrhel" ]; then
+if [ "x$DISTRO" == "xrhel" -o "x$DISTRO" == "xcentos" ]; then
```
Comment (jeclarke) [inline mirror/create_mirror.sh]: Again, no need to prompt for RHEL subscription with centos?

File: mirror/create_mirror_rpm.sh [hunk 3]
```diff
@@ -33,7 +33,10 @@ mkdir -p $RPM_REPO_DIR
 
 cd $RPM_REPO_DIR
 cp /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7 $RPM_REPO_DIR
-cp /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release $RPM_REPO_DIR
+if [ "x$DISTRO" == "xrhel" ]; then
```
Comment (jeclarke) [inline mirror/create_mirror_rpm.sh]: Does this not cause problems when the bootstrap scripts try to import that key?
https://github.com/pndaproject/pnda-aws-templates/blob/develop/bootstrap-scripts/package-install.sh#L33

**Comment by donaldh** [comment]
Will push another branch with fixes and correctly rebased.

**Pull request closed without merging**